msi/release.sh

```diff
@@ -1,17 +1,56 @@
 #!/bin/sh
 
-# Build Git for Windows' .msi files
+me=$(basename $0)
+function showUsage() {
```

Comment:
Ah, that makes sense! And no need to be sorry, really. You are doing really great.